scipy/linalg/tests/test_decomp.py

```diff
@@ -1946,22 +1946,181 @@ def test_qz_double_sort(self):
 
     def test_check_finite(self):
         n = 5
-        A = random([n,n])
-        B = random([n,n])
-        AA,BB,Q,Z = qz(A,B,check_finite=False)
-        assert_array_almost_equal(dot(dot(Q,AA),Z.T), A)
-        assert_array_almost_equal(dot(dot(Q,BB),Z.T), B)
-        assert_array_almost_equal(dot(Q,Q.T), eye(n))
-        assert_array_almost_equal(dot(Z,Z.T), eye(n))
+        A = random([n, n])
+        B = random([n, n])
+        AA, BB, Q, Z = qz(A, B, check_finite=False)
+        assert_array_almost_equal(dot(dot(Q, AA), Z.T), A)
+        assert_array_almost_equal(dot(dot(Q, BB), Z.T), B)
+        assert_array_almost_equal(dot(Q, Q.T), eye(n))
+        assert_array_almost_equal(dot(Z, Z.T), eye(n))
         assert_(all(diag(BB) >= 0))
 
 
+def _make_pos(X):
+    # the decompositions can have different signs than verified results
+    return np.sign(X)*X
+
+
+class TestOrdQZ(TestCase):
+    @classmethod
+    def setupClass(cls):
```

Comment:
Any reason not to change the above two lines to the standard ``def setUp(self):``?